src/Mod/Path/PathScripts/PathAdaptive.py

```diff
@@ -88,6 +83,22 @@ def discretize(edge, flipDirection = False):
 
     return pts
 
+def CalculateCone(radius, alpha):
```

Comment:
Is this function used?

Reply:
This function was already removed in last commit